help/mobile-notifications.md

```diff
@@ -39,6 +35,49 @@ notifications while you are actively using one of the Zulip apps.
 
 {end_tabs}
 
+## Testing mobile notifications
+
+Start by configuring your notifications settings to make it easy to trigger a
+notification.
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{settings_tab|notifications}
+
+1. In the **Mobile** column of the **Notification triggers** table, make sure
+   the **DMs, mentions, and alerts** checkbox is checked.
+
+1. Under **Mobile message notifications**, make sure the **Send mobile
+   notifications even if I'm online** checkbox is checked.
+
+{end_tabs}
+
+Next, test Zulip push notifications on your mobile device.
+
+{start_tabs}
+
+{tab|mobile}
+
+1. [Download](https://zulip.com/apps/) and install the Zulip mobile app if you
+   have not done so already.
+
+1. If you Zulip organization is self-hosted (not at `*.zulipchat.com`),
+   [check](/help/mobile-notifications#enabling-push-notifications-for-self-hosted-servers)
+   whether push notifications have been set up. If they were set up recently,
+   you will need to [log out](/help/logging-out) of your account and [log
+   in](/help/logging-in) again.
```

Comment:
This step is more annoying than is really needed… but I'm not sure I actually have a better alternative, as the alternate way may be too difficult to give foolproof instructions for.

What's really needed is only to cause the app to reload your data from the server from scratch. When it does so, it will also give the server the token, for the server to pass on to our push notifications service, that Apple or Google will require our push notifications service to provide in order to send notifications to your device.

For most users the simplest way to get the app to reload your server data is to just quit the app and restart it. That's the instructions I would definitely give to anyone I was walking through this in person.

The trouble is that I'm not sure how to say "quit the app" in a way that people will find unambiguous. I'd fear many people would interpret "quit the app and restart it" by just navigating to the device's home screen, and then touching the Zulip icon there. But navigating to the home screen leaves the app still running in the background, without quitting it.

Really the high-quality solution here would be to make this instruction unnecessary: the Zulip server should accept the notification token even when notifications aren't set up, so that if the server admin later sets up notifications, it can send to our push notifications service all the accumulated tokens for existing devices and notifications can immediately start working for everyone. But pending that, "log out and log in again" may be the most reliable instructions to give.